content/registries/_index.md

```diff
@@ -13,3 +13,49 @@ weight: 23
 oc patch image.config.openshift.io/cluster -p \
 '{"spec":{"allowedRegistriesForImport":[{"domainName":"my.own.registry.example.com:8888","insecure":true}],"registrySources":{"insecureRegistries":["my.own.registry.example.com:8888"]}}}' --type='merge'
 ```
+
+# Configure custom/insecure registry to search path
```

Comment:
Please add `(4.3)` to the header to clarify it is only valid in 4.3

Reply:
It is 4.x where x is <= 3

Reply:
Could you please add those changes to the PR so I can merge it? Thanks